app/Resources/views/themes/ezplatform/full/bundle_list.html.twig

```diff
@@ -14,16 +14,12 @@
         <div class="bundles-list-search">
             <div class="container">
                 <div class="col-xs-12">
-                    {{ form_start(searchForm) }}
-                        <div class="form-group">
-                            {{ form_widget(searchForm.search, { 'id': 'bundles-list-search-query'} )}}
-                            <div class="input-group-addon">
-                                <button type="submit" class="btn bundles-list-search-button">
-                                    <i class="fa fa-search" aria-hidden="true"></i>
-                                </button>
-                            </div>
-                        </div>
-                    {{ form_end(searchForm) }}
+                    {% if search is defined %}
+                        {% set searchText = search %}
+                    {% else %}
+                        {% set searchText = '' %}
+                    {% endif %}
+                    {{ render_esi(controller('app.controller.bundle:renderSearchForm', {"searchText": searchText})) }}
```

Comment:
Why `render_esi` and not `render`?

Reply:
Because for `render_esi` I'm 100% sure that CSRF token will be regenerated every time. I'm not sure what happens using `render`.

Reply:
sounds good!